src/Microsoft.AspNetCore.Server.HttpSys/FeatureContext.cs

```diff
@@ -446,11 +444,8 @@ ClaimsPrincipal IHttpAuthenticationFeature.User
             set { _user = value; }
         }
 
-        IAuthenticationHandler IHttpAuthenticationFeature.Handler
-        {
-            get { return _authHandler; }
-            set { _authHandler = value; }
-        }
+        [Obsolete("See https://go.microsoft.com/fwlink/?linkid=845470")]
```

Comment:
The obsolete is a bit overkill for an internal property that's implementing an obsolete interface property.